
************************************** Exception Log Start **************************************

****************************************************************************************************

DATE : 

28/12/2020 8:15:24 AM

MESSAGE : 

The process cannot access the file 'D:\Ravi\Personal\MCG\PayrollerManager\Data\Documents\1_photo.jpg' because it is being used by another process.

Stack Trace:

   at System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   at System.IO.File.InternalCopy(String sourceFileName, String destFileName, Boolean overwrite, Boolean checkHost)
   at System.IO.File.Copy(String sourceFileName, String destFileName, Boolean overwrite)
   at PayrollerManager.entity.SaveDocument(String sourcefile, String doctype, Int32 userid) in D:\Ravi\Personal\MCG\PayrollerManager\PayrollerManager\entity.cs:line 119
   at PayrollerManager.frmUsersDetails.SaveDocuments(employeeDetailsModel user) in D:\Ravi\Personal\MCG\PayrollerManager\PayrollerManager\frmUsersDetails.cs:line 337

Source : 

mscorlib

************************************** Exception Log End **************************************

****************************************************************************************************
